style.css

```diff
@@ -1,1041 +1,1413 @@
+    color: #fff;
+    background-color: #6070ff;
+  }
+
+  #submit-button:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .error-message {
+    font-weight: 400;
+    font-style: italic;
+    font-size: 16px;
+    line-height: 28px;
+    color: #ebebff;
+    margin: 14px auto 0 auto;
+    text-align: center;
+  }
+
+  .popup-container {
+    pointer-events: none;
+    width: 100%;
+    top: 50px;
+    left: 0;
+    right: 0;
+    overflow: scroll;
+    max-height: 100%;
+    position: fixed;
+    border: 1px solid #efe1e6;
+    background: rgb(0, 96, 255, 0.3);
+    opacity: 0;
+  }
+
+  .popup-container.show {
+    opacity: 1;
+    pointer-events: auto;
+  }
+
+  .popup {
+    width: calc(80% - 30px);
+    margin: 50px auto;
+    padding: 20px;
+    border-radius: 16px;
+    background: white;
+    box-shadow: 0 8px 16px rgb(64, 83, 252, 0.24);
+    font-family: "Poppins", sans-serif;
+  }
+
+  .cover.show {
+    opacity: 1;
+    pointer-events: auto;
+  }
+
+  .popup.active {
+    top: 100%;
+    transform: scale(1);
+    visibility: visible;
+  }
+
+  .close {
+    width: 20px;
+    height: 20px;
+  }
+
+  .popup-top {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .popup-image {
+    display: block;
+    width: 100%;
+  }
+
+  .popup h4 {
+    margin-left: 0;
+    padding: 0;
+  }
+
+  #popup-content {
+    display: flex;
+    flex-direction: column;
+    gap: 0;
+    padding: 18px;
+  }
+
+  .popup-details {
+    width: 100%;
+  }
+
+  .popup-tech {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  .popup-tech > ul {
+    margin: 20px 0;
+  }
+
+  .buttons {
+    display: flex;
+    gap: 30px;
+    width: 170px;
+  }
+
+  .popup-icon {
+    width: 20px;
+  }
+
+  .buttons > a {
+    color: blue;
+  }
+
+  .popup ul > li {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    color: #6070ff;
+    display: inline-block;
+    list-style-type: none;
+    margin: 0 15px 0 0;
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 5px 10px;
+  }
+
+  .modal {
+    display: none;
+    width: 100%;
+    top: 50px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    overflow: scroll;
+    max-height: 100%;
+    position: fixed;
+    overflow-y: scroll;
+    border: 1px solid #efe1e6;
+    background: rgb(0, 96, 255, 0.3);
+  }
+
+  .popup-card {
+    background-color: white;
+    margin: 30px;
+    display: block;
+    border-radius: 12px;
+    padding: 30px;
+  }
+
+  .popup-img-card {
+    height: 300px;
+    background-repeat: no-repeat;
+    background-size: 95%;
+    margin-left: 15px;
+    display: block;
+  }
+
+  .popup-description {
+    margin-top: 30px;
+  }
+
+  .tag {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    color: #6070ff;
+    display: inline-block;
+    list-style-type: none;
+    margin: 0 15px 0 0;
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 5px 10px;
```

Comment:
- [ ] [**OPTIONAL** ] Kindly, I'll suggest you to change the border of your technologies stacks button to the appropriate color so that your design you'll look like the original design from Figma.
![image](https://user-images.githubusercontent.com/71874425/169665696-ea9299e7-0d21-407d-9634-0c240247f9b2.png)
The border should have the same color with the text inside the border.